students/piotrowski_stanislaw/lesson_03_functions/fibonacci_rec.py

```diff
@@ -0,0 +1,25 @@
+# https://snakify.org/lessons/functions/problems/fibonacci_rec/
+# piotrsta
+
+
+def fib(n):
+    if int(n) == 2 or int(n) == 1:
```

Comment:
za dużo tej konwersji `int(n)`, jeżeli chcesz tego użyć w obrębie tej funkcji, to lepiej wykonaj ja na początku tej funkcji:
`def fib(n):`
`....number = int(n)`
`....if number == 2 ... `
Zabezpiecznie na liczby ujemne lepiej dodac wewnątrz tej funkcji.